api/aboutpage/models/aboutpage.settings.json

```diff
@@ -23,7 +25,7 @@
       "type": "text"
     },
     "headingAboutTechConnect": {
-      "type": "string"
```

Comment:
Ditto for these headings; we don't want them to be very long. Is there a reason they got changed, or is this just a side-effect of something else?